### ISSUE TITLE
AnswerController

### ISSUE THREAD

**Issue by loxiuras** [melsaka/exam-project#6]
https://github.com/melsaka/exam-project/blob/e0f8586f88fd36f5ca4d19463e4e908ad825bb43/app/Http/Controllers/Api/AnswerController.php#L20

Could be set to the following:

`
$answer = Answer::create([
'body' => $request->body,
'status' => $request->status === 'right',
'question_id' => $question->id,
]);
`

**Changes**:
- Create instead of new Model and Save
- Replaced == with ===, please try to used this when you can
- Removed '? true : false', because the result is already a boolean

**Comment by winnie80** [comment]
Is there way to utilize Question model relationship ('answers') to add/create new Answer ?
Something in the line of using attach?
Or this is not possible cause Answer need question_id (not nullable) ?
Please advice ? 
Thanks in advance.

**Comment by loxiuras** [comment]
@winnie80 , yes. $Question is available via the route binding. If the relation is set right, you can do something like 
```
$Question->answer->create( ... );
```
This auto links the `question_id` via the relation.
